gems/pending/util/miq_winrm.rb

```diff
@@ -35,6 +35,18 @@ def run_powershell_script(script)
   rescue WinRM::WinRMAuthorizationError
     $log.info "Error Logging In to #{hostname} using user \"#{username}\"" unless $log.nil?
     raise
+  rescue WinRM::WinRMWMIError => error
+    $log.debug "Error Running Powershell on #{hostname} using user \"#{username}\": #{error}" unless $log.nil?
+    if error.to_s.include? "This user is allowed a maximum number of "
+      $log.debug "Re-opening connection and retrying" unless $log.nil?
+      @executor.close
+      @executor        = nil 
+      @connection      = raw_connect(@username, @password, @uri) if @elevated_runner
+      @elevated_runner = nil
+      retry
```

Comment:
I'm always a bit leary of a retry without a conditional, as it has the potential for an infinite loop.